Core/PARStore.m

```diff
@@ -1458,8 +1468,11 @@ - (void)_sync
     
     // just go through each row (back in time) until all entries are loaded
     NSMutableDictionary *updatedValues = [NSMutableDictionary dictionary];
-    for (NSManagedObject *log in allLogs)
+    for (NSManagedObjectID *logID in allLogObjectIDs)
     {
+        // Get object
+        NSManagedObject *log = [moc objectWithID:logID];
```

Comment:
Ok, found the answer already, yes, you do populate the row cache by default:
http://stackoverflow.com/questions/10532626/how-to-fetch-just-object-ids-but-also-include-the-row-data-in-coredata